types/firestore.d.ts

```diff
@@ -296,6 +296,27 @@ declare namespace FirebaseFirestore {
      * behavior for the underlying BulkWriter.
      */
     bulkWriter(options?: BulkWriterOptions): BulkWriter;
+
+    /**
+     * Creates a new `BundleBuilder` instance to package selected Firestore data into
+     * a bundle.
+     *
+     * @param bundleId. The id of the bundle. When loaded on clients, client SDKs use this id
```

Comment:
In the text part: s/id/ID here and everywhere (including the main sources)

Also drop the period after "bundleId".